sqla_vertica_python/vertica_python.py

```diff
@@ -201,25 +202,49 @@ def get_columns(self, connection, table_name, schema=None, **kw):
           column_name,
           data_type,
           '' as column_default,
-          true as is_nullable
+          true as is_nullable,
+          false as is_identity
         FROM v_catalog.view_columns
         where table_name = '{table_name}'
         {schema_conditional}
         """.format(table_name=table_name, schema_conditional=schema_conditional)
         colobjs = []
-        for row in connection.execute(column_select):
-              colobj = self._get_column_info(
-                  row.column_name, 
-                  row.data_type, 
-                  row.is_nullable,
-                  row.column_default,
-                  (row.column_name in primary_key_columns)
-              )
-              if colobj:
-                  colobjs.append(colobj)
+        column_select_results = list(connection.execute(column_select))
+        for row in column_select_results:
```

Comment:
Using the generator without constructing an intermediate list is preferred, as it was before:

```python
for row in connection.execute(column_select):
```